ui/common/CodeExample.tsx

```diff
@@ -1,14 +1,23 @@
 import { CopyButton } from 'ui'
 
-export default function CodeExample({ code, language }) {
+export default function CodeExample({ code, language, copy }: {code: string; language: string; copy?: boolean | string}) {
+  const copyElement = () => {
+    if(copy === true) {
+      return <CopyButton compact content={code} />
+    } else if(typeof(copy) === 'string') {
```

Comment:
This will also assist us in simplifying the code within this function, by limiting the copyable content to only that within the code prop.